workflow/schema/config.schema.yaml

```diff
@@ -148,7 +148,12 @@ properties:
         items:
           type: string
       intgroup_to_plot:
-        type: string
+        type: 
+          - string
+          - 'null'
+          - array
+        items:
+          type: string

```

Comment:
I don't fully understand this but it seems good

Reply:
Previously the config schema only allowed the intgroup_to_plot param to contain a string (["group"]) but not a list (["group", "index", "plate", "whatever"]). Now either are allowed. It was an oversight when I made the schema.

Reply:
Thanks for the breakdown!